airflow/models/dagrun.py

```diff
@@ -768,7 +764,6 @@ def get_dag(self) -> DAG:
         return self.dag
 
     @staticmethod
-    @internal_api_call
```

Comment:
One more thing. Whenever we see `| ....Pydantic` we should also remove it.

Reply:
Thanks. I removed them